wire-ios-data-model/Source/Model/Conversation/ZMConversation+Participants.swift

```diff
@@ -164,11 +189,13 @@ extension ZMConversation {
 
         case .proteus:
             var action = AddParticipantAction(users: users, conversation: self)
-            action.onResult(resultHandler: completion)
+            action.onResult { result in
+                completion(result)
+            }
```

Comment:
This can stay the same as is it was previously.

Reply:
I believe I shouldn't do anything here

Reply:
reverted